src/components/Navbar.vue

```diff
@@ -3,7 +3,7 @@
     fixed-top
     spaced
     wrapper-class="container"
-    :close-on-click="false"
+    :close-on-click="true"
```

Comment:
I would personally use
```suggestion
    close-on-click
```

Reply:
Great
It also works fine.
Thanks, roiLeo